cfpb/cfgov-refresh
# Upgrade to Capital Framework 10.0.0

Updates Capital Framework to the latest (non-webfont icons) version, and removes some incompatible cruft along the way.

## Removes
- Remove all of `hero.less` file from BAH/OAH, except `.page-intro` class.

## Changes

- Update node on Travis to 10.x
- Update yarn on Travis to 1.15.2
- Update all Capital Framework packages to 10.0.0.
- Convert `u-webfont-medium()` to `font-weight: 500`
- Convert `u-webfont-demi()` to `font-weight: 600`


## Testing

1. Pull branch, run a local server.
1. Check http://localhost:8000/owning-a-home/explore-rates/, http://localhost:8000/owning-a-home/loan-estimate/, http://localhost:8000/owning-a-home/closing-disclosure/ against production. The leading h1 heading of the page will have a very slight change in margin.
2. The external link icon in the notification at the bottom of http://localhost:8000/about-us/blog/fall-2015-rulemaking-agenda/ should not overlap the text around it.
3. The pages noted in https://github.com/cfpb/capital-framework/pull/906#issuecomment-487765459 have icons where they would be expected.
